views/inventory.pug

```diff
@@ -2,4 +2,19 @@ extends layout
 
 block content
 
-  h1 inventory page
+  .section
+    include navbar.pug
+  .section
+    h1 Current Inventory
+
+    form(method='GET', action='/search')
+      input(type='text' placeholder='Apparel...')
+
+  .section
+    ul
+      for result in results
+        li
+          .inventory-item
```

Comment:
This class name holds a lot of information about this section of mark up - I wonder if it would be cleaner to use the enclosing tags instead of an additional div:
```html
ul.inventory
  for result in results
    li.item
      <!-- etc. -->
```
(I should add that this input depends on the way you've decided to structure your CSS as well - if you're going for something BEM like, then the class name `inventory-item` is more appropriate...)